includes/licensing.php

```diff
@@ -912,14 +912,20 @@ public function ajax_deactivate_license() {
 			wp_die();
 		}
 
-		$result   = $this->license_action( $item_type, $item_id, 'deactivate' );
-		$response = new WPBDP_AJAX_Response();
+		$result = $this->license_action( $item_type, $item_id, 'deactivate' );
 
 		if ( is_wp_error( $result ) ) {
-			$response->send_error( sprintf( _x( 'Could not deactivate license: %s.', 'licensing', 'business-directory-plugin' ), $result->get_error_message() ) );
+			if ( 'deactivation-failed' === $result->get_error_code() ) {
+				wp_send_json( array( 'error' => 'deactivation-failed' ) );
+			}
+
+			wp_send_json(
+				array( 
+					'error' => sprintf( _x( 'Could not deactivate license: %s.', 'licensing', 'business-directory-plugin' ), $result->get_error_message() ), 
+				),
+			);
 		} else {
-			$response->set_message( _x( 'License deactivated', 'licensing', 'business-directory-plugin' ) );
-			$response->send();
+			wp_send_json( array( 'message' => _x( 'License deactivated', 'licensing', 'business-directory-plugin' ) ) );
```

Comment:
We can safely remove the `else` here since `wp_send_json` runs the `wp_die()` or `die()` method as well.